mappings/net/minecraft/client/world/DataCache.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_10719 net/minecraft/client/world/DataCache
```

Comment:
I was hesitating on whether or not to call this one a cache, but considering that it can regenerate the stored data at any time, this is probably the right choice.

However, I think this name is way too generic for the two specific usages it has. I like the change from "world" to "context" (since the type parameter implies there could be other kinds of contexts in the future, even though it's only used with `ClientWorld` right now), so maybe `ContextDependentDataCache`?